soh/src/overlays/actors/ovl_Bg_Spot06_Objects/z_bg_spot06_objects.c

```diff
@@ -505,6 +587,13 @@ void BgSpot06Objects_WaterPlaneCutsceneRise(BgSpot06Objects* this, PlayState* pl
     if (this->lakeHyliaWaterLevel >= 0.0001f) {
         this->dyna.actor.world.pos.y = WATER_LEVEL_RAISED;
         this->actionFunc = BgSpot06Objects_DoNothing;
+
+        // On rando, this is used with the water control system switch to finalize raising the water
+        if (gSaveContext.n64ddFlag) {
+            this->lakeHyliaWaterLevel = 0;
+            gSaveContext.eventChkInf[6] |= 0x200; // Set the "raise lake hylia water" flag
```

Comment:
Not a necessity, but it would be nice if we could start using the new macro's defined in z64save when augmenting flags

Reply:
I keep forgetting about those formulas, I wish there was an easy formula to convert these.

Reply:
ok addressed!